### DISCUSSION TITLE
Gitbook Block Style Inconsistencies: Frustrations with Hint Module Styling

### DISCUSSION THREAD

**Discussion by gitacoco** [GitbookIO/community#798] [0. General bugs and questions] [answered]
I’ve noticed a lot of inconsistencies between the block styles in edit mode and in published mode on Gitbook—especially with the HINT block. I really like the lightweight visual style in edit mode, but in published mode, it’s fully filled with blue instead of a subtle style. What I find even more frustrating is that the first paragraph’s font size is smaller than the body text and can’t be adjusted. Having two different font sizes within a single container is really uncomfortable—it not only contradicts the style in edit mode but also doesn’t meet my expectations.

<img width="1971" alt="image" src="https://github.com/user-attachments/assets/9198abf0-fbe3-4bbc-ac80-ba845542d4e5">

**Comment by zenoachtig** [comment, marked as answer]
Hey there, I’m a designer at GitBook 😊 Thanks for your feedback, I'm currently taking a look at inconsistenties between the editor and published content, and I’m hoping to start shipping updates to these soon. Please feel free to point out other inconsistencies, I will put them on the list for me to review.

About the hint block specifically: this is indeed a big disparity, we’ll definitely have to move them closer together. And the font size of the first paragraph seems like a bug, I will investigate.

**Reply by gitacoco** [reply]
Hi zenoachtig, thank you for your response—I really appreciate it. As a heavy GitBook user, your attention means a lot to me. Another inconsistency you might notice in my screenshot is that the markdown headings (Heading 2) at the same level appear in two different font sizes between the editor and published modes.

**Reply by gitacoco** [reply]
Hi zenoachtig, I have identified several additional inconsistencies between the editor and published content. Please check below:
<img width="2056" alt="image" src="https://github.com/user-attachments/assets/8daad384-9f31-40de-998e-adb460e13cbb">

**Reply by zenoachtig** [reply]
Hi @gitacoco, thanks again for your detailed report of these inconsistencies. Many of them have to do with tables – I will review that component soon.

Meanwhile, I wanted to let you know that I have created https://github.com/GitbookIO/gitbook/pull/2579 to give the hint blocks a more modern styling. I will implement these same styles in the editor soon, so the two are more closely aligned.

**Reply by zenoachtig** [reply]
Hi @gitacoco, final update on this: the new designs for hint blocks are now rolling out to both the editor and published documentation. You should be seeing it soon 😊

**Reply by ahcsco** [reply]
Hi, there is no option to change font in editor UI. Do you plan to have that possibility to be able to choose a font style in editor to have it same as published site font?
